File: 2021 March/2021.03.05 완전탐색/kakaru_모의고사.js
function solution(answers) {
    var answer = [];
    var supo1 = [1,2,3,4,5];
    var supo2 = [2,1,2,3,2,4,2,5];
    var supo3 = [3,3,1,1,2,2,4,4,5,5];
    var su = [0,0,0];
    for(var i = 0; i <answers.length; i++){
        if(supo1[i%5]==answers[i]){
            su[0]++;
        };
        if(supo2[i%8]==answers[i]){
            su[1]++;
        };
        if(supo3[i%10]==answers[i]){
            su[2]++;
        };
    };
    var max = Math.max(su[0], su[1], su[2])
    for(var i = 0; i < 3; i++){
        if(su[i]== max){
            answer.push(i+1);
        }
    }
    return answer;
}

File: 2021 March/2021.03.05 완전탐색/kakaru_모의고사.py
def solution(answers):
    answer = []
    supo1 = [1,2,3,4,5]
    supo2 = [2,1,2,3,2,4,2,5]
    supo3 = [3,3,1,1,2,2,4,4,5,5,3,3,1,1,2,2]
    cnt =[0,0,0]
    for i in range(len(answers)):
        if answers[i]==supo1[(i%5)]:
            cnt[0]+=1
        if answers[i]==supo2[(i%8)]:
            cnt[1]+=1
        if answers[i]==supo3[(i%10)]:
            cnt[2]+=1
    
    for i in range(3):
        if cnt[i]==max(cnt):
            answer.append(i+1)
    return answer
